Ks/readme (#51)
team-soapstone/front-end-capstone: README.md

![Soapstone Banner](./soapstoneBanner.png)

# Project Catwalk - Soapstone Group

## Table of Contents

1. [General Info](#general-info)
2. [Demo](#demo)
3. [Technologies Used](#technologies-used)
4. [Contributors](#contributors)

## General Info

> Soapstone Group is a group of like-minded individuals that came together deliver based off of client needs. For this particular product, we are creating a modernized client-facing retail web-portal, called Project Catwalk. This retail portal will provide the following function upon it's initial release and deployment:

Reviewer: ```suggestion
Soapstone Group is a group of like-minded individuals that came together to deliver based off of client needs. For this particular product, we are creating a modern client-facing retail web-portal, called Project Catwalk. This retail portal will provide the following function upon its initial release and deployment:
```

Author: fixed!

**Overview**
* An image gallery
* Information about the current product
* A selector to choose between varying styles
* Add to cart functionality

**Questions and Answers**
* A list of all questions asked by users
* A search functionality to filter by questions
* Ability to ask and answer a question

**Reviews**
* A list of all reviews given by users
* A breakdown of the ratings and the ability to filter reviews
* An additional breakdown of comfort and fit of the product
* Ability to provide a review
## Demo

### Overview
![Overview](./demo/overviewDemo.gif)

### Questions & Answers
![Questions and Answers](./demo/questionsDemo.gif)

### Ratings & Reviews
![Ratings and Reviews](./demo/ratingsDemo.gif)

### Changing Product with Search Bar
![Search Bar](./demo/changeProductDemo.gif)
## Technologies Used

> Please note that the below are a list of all major dependencies used in the project. There could be various adapters and minor dependencies needed. Please ensure to install the necesary modules with NPM.

Reviewer: ```suggestion
Please note that below is a list of all major dependencies used in the project. There could be various adapters and minor dependencies needed. Please ensure to install the necesary modules with NPM.
```

Author: fixed!
